docs/api.md

```diff
@@ -203,9 +218,103 @@ assert response.status_code == 200
 ```
 
 
-### Report
+## Report Endpoint
+
+Two types of requests can be made:
+
+- To request a Report for an AOI defined by OQT provide following parameters: `name`, `dataset` and `featureId`
+- To request a Report for a custom AOI provide following parameters: parameters: `name` and `bpolys`
+
+Depending on the input, the output will be a GeoJSON Feature or FeatureCollection with the results of the Report and each Indicator as properties. The Feature properties of the input GeoJSON will be preserved if they do not collide with the properties set by OQT.
+
+### Parameters
+
+- `name`: Name of the Rerpot
+- `dataset`: Name of the dataset containing bounding polygons
+- `featureId`: Identifier of the feature in the dataset
+- `bpolys`: Bounding polygon(s) as GeoJSON Feature or FeatureCollection object. The Geometry of those objects has to be Polygon or MultiPolygon.
+- `includeSvg` (boolean): Include a SVG string of a figure displaying the Indicator
+results
+- `includeHtml` (boolean): Include a HTML string with the Indicator results
+
+Parameters are in part mutual exclusive (E.g. either `dataset` and `featureId` or `bpolys`). If the response contains multiple errors probably not all required parameters and/or mutual exclusive parameters has been provided.
+
+### A Note on the Difference of `GET` and `POST` Requests
+
+To the endpoint `/report` a request can be made using either the `GET` or `POST` method.
+For `GET` requests to this endpoint only the parameters `dataset` and `feature_id` are supported.
+That means that it is not possible to create a Report for a custom AOI using the `bpolys` parameter by making a `GET` request to the API.
+
+Using the `POST` method all types of requests (see above) and parameters are supported.
+
+### Request an Report for an OQT AOI using Python and `requests` Library
+
+GET request:
+
+```python
+import requests
+
+url = "https://oqt.ohsome.org/api/report"
+parameters = {
+    "name": "SimpleReport",
+    "dataset": "regions",
+    "featureId": 3,
+    "includeSvg": False,
+    "includeHtml": False,
+}
+response = requests.get(url, params=parameters)
+assert response.status_code == 200
```

Comment:
see above comment about the assert

Reply:
Removed the assert statement as it is not necessary.